tests/rng_tools/README.md

```diff
@@ -0,0 +1,25 @@
+# RNG Tools
+Test application for the RNG module.
+
+## Supported commands
+* distributions [N] &mdash; Take N samples and print a bit distribution graph on the terminal.
```

Comment:
Can you either use another symbol (e.g. `:`) or the unicode character `—` instead of `&mdash;`? GitHub should be able to handle that and this way it is also easier to read offline ;-)

Reply:
Old habit, will fix :-)